usmqe_tests/rpm/packagelist.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf8 -*-
+
+tendrl_packages = [
+    "tendrl-alerting",
+    "tendrl-api",
+    "tendrl-api-httpd",
+    "tendrl-ceph-integration",
+    "tendrl-commons",
+    "tendrl-dashboard",
+    "tendrl-gluster-integration",
+    "tendrl-node-agent",
+    "tendrl-node-monitoring",
+    "tendrl-performance-monitoring",
+    ]
```

Comment:
I think, it might be worth to be able to notice removed or newly added package. (Removed package should be currently detected, but I'm not sure about newly added?)
Otherwise I haven't strong opinion about the way how to define/get the package list (statically/dynamically).